crates/rpc/bridge/src/lib.rs

```diff
@@ -15,3 +21,39 @@ pub trait StrataBridgeControlApi {
     #[method(name = "uptime")]
     async fn get_uptime(&self) -> RpcResult<u64>;
 }
+
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "stratabridge"))]
+#[cfg_attr(feature = "client", rpc(server, client, namespace = "stratabridge"))]
+pub trait StrataBridgeMonitoringApi {
+    /// Get all bridge operator IDs
+    #[method(name = "bridgeOperators")]
+    async fn get_bridge_operators(&self) -> RpcResult<Vec<OperatorIdx>>;
+
+    /// Query operator status (Online/Offline)
+    #[method(name = "operatorStatus")]
+    async fn get_operator_status(&self, operator_idx: OperatorIdx) -> RpcResult<RpcOperatorStatus>;
```

Comment:
This might need a new P2P command.
I don't know how to circumvent this.
Which is fine, we should have that.
Cc @Rajil1213